Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/DonorModule.kt

```diff
@@ -0,0 +1,32 @@
+package de.rki.coronawarnapp.datadonation.analytics.modules
+
+interface DonorModule {
+
+    suspend fun startDonation(request: Request): Contribution
```

Comment:
very minor: maybe name it `initiateDonation` instead, to clarify it's the first thing being called.